### ISSUE TITLE
how to make title or other elements below the header elements

### ISSUE THREAD

**Issue by vanlong2712** [react-native-training/react-native-elements#655]
![image](https://user-images.githubusercontent.com/21197310/31572102-51ddd610-b0c8-11e7-8252-532cdb335653.png)

```
import React, { Component } from 'react';
import { View } from 'react-native';
import { Header, Text } from 'react-native-elements';

class MultiAnswer extends Component {
  render() {
    return (
      <View style={{ flex: 1 }}>
        <Header
          centerComponent={{
            text: 'Multi Answer',
            style: { fontSize: 20, color: 'blue' }
          }}
        />
        <Text h3>Pick 1 to 3</Text>
      </View>
    );
  }
}
```

**Comment by antoinema** [comment]
You need to use `marginTop` to move your content down. You can have a look at this discussion https://github.com/react-native-training/react-native-elements/issues/568#issuecomment-330585188 to understand why

**Comment by iRoachie** [comment]
Hey @vanlong2712 a fix will be released in the next version that makes the header work as you would expect and items will go beneath instead of overlap. Feel free to use the `next` branch in the meantime.

**Comment by iRoachie** [comment]
Fixed in v18.0.0
